### PR TITLE
[TSD] Add annotations for  ORA2_FILEUPLOAD_BACKEND

### PR DESCRIPTION
**Pull request by morenol**
**TL;DR -** Add annotations for ORA2_FILEUPLOAD_BACKEND django setting but does not affect behavior.

**What changed?**

Added annotations for ORA2_FILEUPLOAD_BACKEND django setting as part of the doc-a-thon event!

**Testing Instructions**

[ How should a reviewer test this PR? ]
You should review that the annotations follow the conventions of openedx 
https://openedx.atlassian.net/wiki/spaces/AC/pages/2248409171/Toggles+and+Settings+Doc-a-thon+2021#1.-Short-Background-Reading

**Reviewer Checklist**

Collectively, these should be completed by reviewers of this PR:

- [ ] I've done a visual code review

FYI: @edx/masters-devs-gta

### CODE REVIEW

**Comment by edx-webhook** [comment]
Thanks for the pull request, @morenol! I've created [OSPR-5567](https://openedx.atlassian.net/browse/OSPR-5567) to keep track of it in JIRA, where we prioritize reviews. Please note that it may take us up to several weeks or months to complete a review and merge your PR.

Feel free to add as much of the following information to the ticket:
- supporting documentation
- [Open edX discussion forum threads](https://discuss.openedx.org/)
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).


Please let us know once your PR is ready for our review and all tests are green.

**Comment by natabene** [comment]
@morenol Thank you for your contribution. @robrap Please review.

**Review by robrap — APPROVED** [review]

**Comment by robrap** [comment]
Thanks @morenol.  Looks great.  Thanks for providing the options. :)

**Comment by edx-webhook** [comment]
@morenol 🎉 Your pull request was merged!

Please take a moment to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=edx/edx-ora2&entry.867055334=https://github.com/edx/edx-ora2/pull/1579&entry.1484655318=https://github.com/morenol&entry.752974735=2021-01-27+17:46&entry.1917517419=2021-01-27+21:23&entry.2133058324=Yes) so we can improve your experience in the future.

File: openassessment/fileupload/backends/__init__.py [hunk 1]
```diff
@@ -7,7 +7,10 @@
 
 
 def get_backend():
-    # Use S3 backend by default (current behaviour)
+    # .. setting_name: ORA2_FILEUPLOAD_BACKEND
+    # .. setting_default: s3
```
Comment (robrap) [inline openassessment/fileupload/backends/__init__.py:11]: @morenol: After merging, I realized that the default should probably include quotes like `'s3'`.  Thoughts?  Would you mind updating?

Comment (morenol) [reply openassessment/fileupload/backends/__init__.py:11]: Yes. I'll create a new PR.
